babylonhealth/terraform-provider-azuredevops
# Merge in v0.1.5

This isn't going to cut it; see #9 for an alternatve approach

- serviceendpoint_servicefabric
- working certificate
- add aad
- thumbprints and common names
- refactor
- tests
- rename file
- correct imports
- spelling
- docs
- update provider test
- fix CI tools install
- fix group scope not set
- CI update
- remove github.com/hashicorp/terraform
- remove github.com/hashicorp/terraform
- add none auth type
- terraform-sdk 1.17.2
- terraform-sdk 1.17.2
- change crlf to lf
- remove unused command
- update docs and validation
- update validation
- branch policy status check
- doc update
- v0.1.5 changelog
- doc update
- policy type
- new resource repository policy - author email pattern
- update document
- allow updating azurerm service endpoint
- fix lint issue
- azurecr meta data cannot be updated
- update changelog
- update changelog
- new resource azuredevops_repository_policy_file_path_pattern
- CHANGELOG.md updaate
- pipeline timeout
- pipeline timeout
- preprare v0.1.5
- fmt

## All Submissions:

* [ ] I ran lint checks locally prior to submission.
* [ ] Have you checked to ensure there aren't other open PRs for the same update/change?

## What about the current behavior has changed?



Issue Number:

## Does this introduce a change to `go.mod`, `go.sum` or `vendor/`?




## Does this introduce a breaking change?




## Any relevant logs, error output, etc?

(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)


## Other information
